src/test/resources/bundles/userCardExamples3/template/usercard_task.handlebars

```diff
@@ -123,4 +120,11 @@
         };
 
     }
-</script>
\ No newline at end of file
+</script>
+
+ <style>
+  #hours {
+    text-align:center; 
+  }
+
+  </style>
```

Comment:
To be consitent with other templates just put the style directly in the input :  <input style="text-align:center"  ...